### DISCUSSION TITLE
0x00000010 error on fresh SD card setup of v15.2.4

### DISCUSSION THREAD

**Discussion by z-master42** [jomjol/AI-on-the-edge-device#2408] [General]
### The Problem

After following web install firmware flash and then manual SD card setup:

> We have one or more critical errors:
> 0x00000010

When comparing with the error code description page, I noticed that `version.txt` is missing in `/sdcard/html/`. I manually added from the [update.zip](https://github.com/jomjol/AI-on-the-edge-device/releases/tag/v15.2.4) and this fixed the error.

### Version

15.2.4

### Logfile

```shell
Not available
```

**Comment by caco3** [comment]
where id you get the files from you put onto the SD-Card? You cant take them directly from the GIT repo.

**Reply by z-master42** [reply]
I used the source code zip from the release page to get the files which should be on the SD card. This is what the instructions say that I used. I will point out to the creator of this manual that it is different and correct in the official manual.

**Reply by caco3** [reply]
Please follow the official documentation on https://jomjol.github.io/AI-on-the-edge-device-docs/Installation/#3-sd-card
